### PR TITLE
Additional fixes for Marketo connector

### PR DESCRIPTION
**Pull request by pdulapalli**
* Fixes a date formatting issue that broke normalization
* Revises what fields are extracted for Marketo Leads
    * Now includes whatever Salesforce IDs are possibly available
    * Filtering out most other fields for safety -- in the CSV export some have unescaped newline characters --> breaks parsing
 * Add Java version file to make it easier to run locally w/ [Jenv](https://www.jenv.be/) (recommended by Airbyte docs)

### CODE REVIEW

File: airbyte-integrations/connectors/source-marketo/source_marketo/source.py [hunk 1]
```diff
@@ -447,6 +447,12 @@ def request_params(
     def parse_response(self, response: requests.Response, stream_state: Mapping[str, Any], **kwargs) -> Iterable[Mapping]:
         for record in super().parse_response(response, stream_state, **kwargs):
             record["updatedAt"] = record["updatedAt"][:-5]  # delete +00:00 part from the end of updatedAt
+
+            # Remove non parser-friendly suffix from createdAt
+            created_at = record.get("createdAt", "")
+            if created_at[-5:] == "+0000":
```
Comment (pacificsky) [inline airbyte-integrations/connectors/source-marketo/source_marketo/source.py:453]: Curious why this logic is subtly different from similar logic at line 449 for updatedAt?

**Review by pacificsky — APPROVED** [review]

File: airbyte-integrations/connectors/source-marketo/source_marketo/source.py (hunk 1 above)
Comment (pdulapalli) [reply airbyte-integrations/connectors/source-marketo/source_marketo/source.py:453]: It fundamentally does the same as the `updatedAt` logic, except I don't think `createdAt` exists in all of the Marketo data sources. Also, I want to make sure that we only replace the suffix if it's problematic and not just slice off the last 5 characters. 

tl;dr: Erring on the "better safe than sorry" side for this one

Comment (pacificsky) [reply airbyte-integrations/connectors/source-marketo/source_marketo/source.py:453]: yeah but why is the format of the timestamp youre chopping off different?

Comment (pdulapalli) [reply airbyte-integrations/connectors/source-marketo/source_marketo/source.py:453]: In a few instances, I didn't see the `+0000` in the source value, so those cases might chop off too much.
